save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -54,6 +55,7 @@ external interface OrganizationViewState : State {
  * A Component for owner view
  */
 class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(false) {
+    private var organization = Organization("", null, null, null)
```

Comment:
I believe it should be in state, otherwise data will not update after it's fetched in `componentDidMount`

Reply:
The organization is used only for the title name, so it is not necessary to add it to the state

Reply:
Yes, but request is sent asynchronously, so there is no guarantee that organization.name will be set with the correct value when title is rendered

Reply:
Ok. Changed